chain/chain/src/test_utils.rs

```diff
@@ -154,17 +159,27 @@ impl KeyValueRuntime {
         self.root.hash
     }
 
+    fn get_block_header(&self, hash: &CryptoHash) -> Result<Option<BlockHeader>, Error> {
```

Comment:
this is really weird. why don't you just use ChainStore here to load headers instead of loading them directly?

Reply:
Because it doesn't have access to `ChainStore` today